gtsfm/averaging/rotation/shonan.py

```diff
@@ -37,7 +37,7 @@ def __init__(self) -> None:
         """
         Note: `p_min` and `p_max` describe the minimum and maximum relaxation rank.
         """
-        self._p_min = 5
+        self._p_min = 30
```

Comment:
Why this change?

Reply:
so when we use a robust cost like Huber, we don't climb the Riemannian staircase from SO(5) to SO(30). Instead, we choose a single relaxation rank p and solve for the SO(p) matrices. So pmin must equal pmax

Reply:
Hmmm. Let’s discuss tomorrow